03/ukol17.py

```diff
@@ -0,0 +1,15 @@
+#Napiš program, který se pětkrát zeptá na číslo a nejmenší zadané číslo vypíše.
+
+first_number = int(input('Zadej prvni cislo: '))
+second_number = int(input('Zadej prvni cislo: '))
+third_number = int(input('Zadej prvni cislo: '))
+fourth_number = int(input('Zadej prvni cislo: '))
+fifth_number = int(input('Zadej prvni cislo: '))
+
+numbers = [first_number, second_number, third_number, fourth_number, fifth_number]
+
+numbers.sort()
+
+
+print(f'Nejmensi cislo ze seznamu je {numbers[:1]} .')
```

Comment:
Funguje to - a koukám, žes to dala aj s listama ještě před tím, než jsme přednášku o listech měli 😄.

A koukám, že používáš aj `list.sort()`. Takže pokud už metody znáš, tak by se to také dalo přepsat tak, že by to bylo celé kompaktnější:

```
cisla = []
while i in range(5):
  vstup = input(f"Zadej {i + 1}. číslo: ")
  cisla.append(vstup)

print(f"Nejmenší číslo jest: {min(cisla)}")
```

Ve výrazech v chlupatých závorkách `{...}` v f-stringu lze normálně volat třeba i funkce. V tomhle případě jsme použili vestavěnou funkci `min()`, které podstrčíme nějaký ***iterable*** _(iterable je něco, přes co se dá iterovat - tedy např. list, string, tuple, dokonce aj slovník nebo set)_ - a z funkce se pak vrátí prvek, který má v celém iterable nejnižší hodnotu.

https://docs.python.org/3/library/functions.html#min